### ISSUE TITLE
Allow keypaths to preserve order

### ISSUE THREAD

**Issue by mucmch** [fabiocaccamo/python-benedict#404]
Given a benedict dictionary d, invoking d.keypaths() will return all paths in a sorted order.

There are cases where this is an undesired behavior and is caused by `kps.sort()` in benedict.core.keypaths.py.

I would propose a new parameter "sorted" to keypaths that allows to disable this line of code.

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/fabiocaccamo) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/fabiocaccamo/python-benedict/issues/404">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/fabiocaccamo/python-benedict/issues/404/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/fabiocaccamo/python-benedict/issues/404/pledge.svg">
</picture>
</a>

**Comment by fabiocaccamo** [comment]
@mucmch I agree with adding a "sorted" parameter to keypaths method (defaults to `True` for backward compatibility). Do you want to submit a PR with a couple tests?

**Comment by mucmch** [comment]
Did a simple implementation, but do not manage to commit to Github. I attached a patch file.

[keypaths_order.patch](https://github.com/fabiocaccamo/python-benedict/files/15162276/keypaths_order.patch)

**Comment by kotfu** [comment]
New user of benedict here, and I could benefit from this "sorted" parameter. I'll submit a PR shortly.

**Comment by fabiocaccamo** [comment]
@kotfu thank you very much for the PR!
@mucmch parameter available in `0.34.1`.
